roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +102,32 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ edpm_nova_config_sync_dest }}"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
+    mode: "push"
+    owner: true
+    perms: true
+  delegate_to: "{{ inventory_hostname }}"
```

Comment:
OK, we need this to run rsync between two directories on the EDPM node as by default rsync runs in the aee pod.

Reply:
we only need to do this because we need to flatten multiple direcoties into one.

we could technically flatten them in the AEE container and then rsync the result instead.
that might make more sense.

Reply:
I agree, this would be a better path to take

Reply:
done, now rsync is between ansible container and edpm_node